packages/odyssey/src/scss/base/_typography-list.scss

```diff
@@ -50,6 +50,11 @@ dl:not([class]) {
   max-width: $max-line-length;
   margin: 0 0 $spacing-s 0;
   padding: 0;
+  font-size: $size-body-sentence;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
```

Comment:
How did this come in here?

Reply:
This was pushed to the release branch as a UI fix for the PR - Design had signed off on something different than what got shipped.